client/podman/container_mock.go

```diff
@@ -350,6 +366,14 @@ func (r *RestClientMock) HandleContainerCreate(operation *runtime.ClientOperatio
 		RestartCount:  int(spec.RestartRetries),
 		CreatedAt:     time.Now(),
 	}
+	if spec.ResourceLimits != nil {
+		if spec.ResourceLimits.CPU != nil {
+			c.Cpus = int(spec.ResourceLimits.CPU.Quota / 100000)
+		}
+		if spec.ResourceLimits.Memory != nil {
+			c.MemoryBytes = spec.ResourceLimits.Memory.Limit
```

Comment:
Minor naming: To me, ResourceLimits.Memory.Limit is a MaxMemoryBytes or MemoryLimitBytes. This may just be my lack of familiarity with the podman APIs, or my inference from Kubernetes. MemoryBytes sounds like the current memory rather than a limit.

Reply:
Agreed. I will update it.